### PR TITLE
Update internal docs

### PR DESCRIPTION
**Pull request by anayden**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/.neuro/live.yml [hunk 1]
```diff
@@ -44,14 +44,14 @@ jobs:
     action: gh:neuro-actions/remote_debug@master
     args:
       image: $[[ images.myimage.ref ]]
-      volumes_data_ref: $[[ volumes.data.remote ]]
-      volumes_code_ref: $[[ volumes.code.remote ]]
-      volumes_config_ref: $[[ volumes.config.remote ]]
-      volumes_results_ref: $[[ volumes.results.remote ]]h
+      volumes_data_remote: $[[ volumes.data.remote ]]
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/.neuro/live.yml:47]: Do not forget to update also [this](https://github.com/neuro-actions/remote_debug/blob/master/.neuro/live.yaml) guy.

File: {{cookiecutter.project_slug}}/HELP.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ Follow the instructions below to set up the environment on Neuro and start a Jup
 ### Setup development environment 
 
 ```shell
-make setup
+neuro-flow build myimage
 ```
 
 * Several files from the local project are uploaded to the platform storage (namely, `requirements.txt`,  `apt.txt`, `setup.cfg`).
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/HELP.md:33]: Pls, update that the entire `context` folder is uploaded.

File: {{cookiecutter.project_slug}}/HELP.md [hunk 3]
```diff
@@ -50,4 +50,133 @@ neuro-flow run jupyter
 neuro-flow kill jupyter
 ```
 
-* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `make download-notebooks` to download them to the local `notebooks/` directory.
+* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage.
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/HELP.md:53]: we could preserve it, neuro-flow could do that
```suggestion
* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `neuro-flow download notebooks` to download them to the local `notebooks/` directory.
```

File: {{cookiecutter.project_slug}}/HELP.md [hunk 4]
```diff
@@ -50,4 +50,133 @@ neuro-flow run jupyter
...
+### Uploading to the Storage via Web UI
+
+On local machine, run `neuro-flow run filebrowser` and open the job's URL on your mobile device or desktop.
+Through a simple file explorer interface, you can upload test images and perform file operations.
+
+### Uploading to the Storage via CLI
+
+On local machine run `neuro-flow mkvolumes`. This commands creates storage folders for all defined volumes. You only need to run this once.
+
+After storage folders have been created run `neuro-flow upload data` from local machine as well. This command pushes local files stored in ./data into storage:{{ cookiecutter.project_slug }}/data mounted to your development environment's /project/data.
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/HELP.md:75]: Let's add a bit more info:
```suggestion
After storage folders have been created run `neuro-flow upload data` from the local machine as well. This command pushes local files stored in ./data into storage:{{ cookiecutter.project_slug }}/data mounted to your development environment's /project/data.

You can upload (or download) every folder for which the `local` parameter is specified in [live.yml file](./.neuro/live.yml).
```

File: {{cookiecutter.project_slug}}/HELP.md [hunk 5]
```diff
@@ -50,4 +50,133 @@ neuro-flow run jupyter
...
+
+### Uploading data to the Job from Google Cloud Storage
+
+Google Cloud SDK is pre-installed on all jobs produced from the Base Image.
+
+Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-gcp) explaining how to start using GCP with the template.
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/HELP.md:81]: these docs will be updated to refer `neuro-extras` usage later?

File: {{cookiecutter.project_slug}}/.neuro/live.yml [hunk 6]
```diff
@@ -44,14 +44,14 @@ jobs:
     action: gh:neuro-actions/remote_debug@master
     args:
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/.neuro/live.yml:45]: What if we expose the local port (2211) as a parameter with a default 2211 value, wdyt?

File: {{cookiecutter.project_slug}}/HELP.md [hunk 7]
```diff
@@ -50,4 +50,133 @@ neuro-flow run jupyter
...
+AWS CLI is pre-installed on all jobs produced from the Base Image.
+
+Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-aws) explaining how to start using AWS with the template.
+
+### Run development job
+
+If you want to debug your code on GPU, you can run a sleeping job via `neuro-flow run remote_debug`, which will also open a shell to the job. You can also see job logs via `neuro-flow logs remote_debug`. The job forwards your local port 2211 to its port 22for remote debugging.
+Please find instructions on remote debugging via PyCharm Pro in the [documentation](https://neu.ro/docs/remote_debugging_pycharm). 
+
+Please don't forget to kill your job via `make kill remote_debug` not to waste your quota!
```
Comment (YevheniiSemendiak) [inline {{cookiecutter.project_slug}}/HELP.md:94]:
```suggestion
Please don't forget to kill your job via `neuro-flow kill remote_debug` not to waste your quota!   
```

**Review by YevheniiSemendiak — APPROVED** [review]
Thank you for enhancing docs for one of the most important parts of our platform! 👍 🚀 
Below I've left some comments that might be useful but in general all LGTM!

File: {{cookiecutter.project_slug}}/HELP.md (hunk 5 above)
Comment (anayden) [reply {{cookiecutter.project_slug}}/HELP.md:83]: Yes, absolutely. It is in the plans: as soon as `neuro-flow` supports multiple volumes/envvars passing to custom actions

File: {{cookiecutter.project_slug}}/.neuro/live.yml (hunk 6 above)
Comment (anayden) [reply {{cookiecutter.project_slug}}/.neuro/live.yml:45]: Good idea. Updated corresponding `action.yaml`

**Pull request merged**